docs/source/index.rst

```diff
@@ -20,6 +20,7 @@ This guide will explain how to set up Quark, use it, and customize it.
 
    install_index
    quark_script
+   quark_mit
```

Comment:
I suggest to use the full name of this project.
Please use "quark mit program" instead of just "quark mit"